recipes/simmate/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "simmate" %}
+{% set version = "0.0.0.dev3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # Once my github is made public, I can use archives from the releases with...
+  # url: https://github.com/jacksund/simmate/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f5882ff5895550f50a70b8e798342faf5376fdd113cb47a6c4c5a10d916e775f
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+      - simmate = simmate.command_line.base_command:simmate
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    # One of my dependencies (pymatgen --> spglib) requires this. Should I still
+    # include this compiler here?
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy ==1.21.4
```

Comment:
Yep, I agree this needs to be fixed. My test suite is sorely lacking in the repo right now, so I'm starting strict with the dependencies and loosening as implement my tests. 

For my team's alpha testing, I'm making things really strict as I know it will be the only thing in their conda environments.

Reply:
Yep, that's okay. I didn't notice that this was a `.dev` release at first. So it should be fine.

Reply:
Sweet, that's good to hear. I'll mark this as resolved then.